src/styles/blocks/card.scss

```diff
@@ -0,0 +1,141 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid $c-light;
+  border-radius: 5px;
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+
+  &__content {
+    display: flex;
+    align-content: stretch;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: stretch;
+
+    width: 100%;
+  }
+
+  &__prodact-img {
+    height: 134px;
+  }
+
+  &__monitor-title {
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $c-main-text;
+
+    display: flex;
+  }
+
+  &__monitor-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $c-second-text;
+
+    display: block;
+
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+
+    margin-top: 16px;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    display: flex;
+    align-content: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
```

Comment:
It's totally up to you but you can use variables of font values as well as for colors. It is very useful in bigger tasks.